### ISSUE TITLE
SIMPLE_IDENTIFIER is not assigned when it has to be

### ISSUE THREAD

**Issue by EgorBu** [bblfsh/python-driver#22]
Hi,
I tried to extract UAST from python code and noticed when you define a function:
`def a(b, c): ...`
the node for this function will have roles: `‘FUNCTION_DECLARATION_BODY’`, `‘FUNCTION_DECLARATION_RECEIVER’`, but not `'SIMPLE_IDENTIFIER'`.
In the documentation it's mentioned:
```
// SimpleIdentifier is the most basic form of identifier, used for variable
// names, functions, packages, etc.
```
I think that this node should have `'SIMPLE_IDENTIFIER'` role.

**Comment by juanjux** [comment]
Should be fixed by PR #23 when merged.

**Comment by smola** [comment]
Fixed.
